web-ui/src/pages/BirthdayReportPage.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useContext, useState } from "react";
```

Comment:
This file is like `AnniversaryReportPage`, but deals with birthdays instead of anniversaries.